src/components/popup/Popup.tsx

```diff
@@ -67,8 +67,10 @@ export const Popup = React.forwardRef<HTMLDivElement, PopupProps>(
                 tabIndex={-1}
                 role="dialog"
                 aria-modal="true"
-                onClick={backdropClosable && onHide ? onHide : undefined}
                 ref={ref}
+                onMouseDown={(e: React.MouseEvent<HTMLDivElement, MouseEvent>) =>
```

Comment:
@OlhaNikoliuk Hello!

Is there a specific reason why you use `onMouseDown` instead of `onClick`?